freestyle/shared/src/test/scala/freestyle/implicits.scala

```diff
@@ -0,0 +1,35 @@
+package freestyle
+
+import cats.implicits._
+import cats.Id
+import org.scalatest.{Matchers, WordSpec}
+
+class implicitsTests extends WordSpec with Matchers {
+
+  "Implicits" should {
+
+    import algebras._
+    import freestyle.implicits._
+
+    "make monad implicitly available" in {
+      type G[A] = FreeS[SCtors1.Op, A]
+      implicitly[cats.Monad[G]].isInstanceOf[cats.Monad[G]]
```

Comment:
The scalatest 'compile' check doesn't seem to play well with kind-projector compiler plugin. I have left the type alias.